.github/workflows/build.yaml

```diff
@@ -355,6 +363,12 @@ jobs:
         timeout-minutes: 200
         runs-on: macos-latest
         if: github.actor != 'restyled-io[bot]'
+        
+        env:
+            # NOTE: zap version should be kept in sync with the image used by 
+            # Docker, currently connectedhomeip/chip-build:0.6.31 
+            ZAP_VERSION: v2023.01.09-nightly
+            ZAP_INSTALL_PATH: /usr/local/zap/zap-v2023.01.09-nightly
```

Comment:
Can this use ZAP_VERSION instead of hardcoding, to minimize the number of things that need to be changed on ZAP updates?  Same for other places that use this pattern.

Reply:
For now a script automates things.
Realistically this path does NOT need to be versioned so I could remove that, but if we have a script we do not care as much right?

Reply:
If we have a script, then this is fine.